Lib9c/Model/Character/CharacterBase.cs

```diff
@@ -391,7 +395,9 @@ public int GetCooldown(int skillId)
 
         public void ReduceCooldown()
         {
+#pragma warning disable LAA1002
             if (!_skillsCooldown.Any())
+#pragma warning restore LAA1002
```

Comment:
:+1: I am going to fix this false alarm: <https://github.com/planetarium/libplanet/issues/1048>.